doc/index.rst

```diff
@@ -27,7 +27,6 @@ Welcome to the Cabal User Guide
 
    cabal-package-description-file
    cabal-project-description-file
-   version-control-fields
```

Comment:
Of all the cabal-related things I do, adding a `source-repository-package` to a `cabal.project` is the most frequent when maintaining a project. To be able to look up these fields with one click is a convenience I'd not like to loose.